docs/ReleaseGuide.md

```diff
@@ -48,4 +54,8 @@ The release of the `python-bindings` repository is made directly from a release
 
    For more details refer to https://github.com/precice/python-bindings/issues/109 and https://github.com/python-versioneer/python-versioneer/issues/217.
 
-8. Update Spack package (refer to `python-bindings/spack/README.md`).
+9. *Temporarily not maintained* Update Spack package (refer to `python-bindings/spack/README.md`).
+
+## Pre-release
+
+A pre-release branch is never merged into a master branch. After creating the branch and drafting a release, directly hit the "Publish release" button in your Release Draft. Now you can check the pre-release artifacts (e.g. release on [PyPI](https://pypi.org/project/pyprecice/#history)) of the release. No further action is required for a pre-release.
```

Comment:
```suggestion
The purpose of a pre-release is to test the release workflow (packaging, upload to PyPI). After drafting a release, directly hit the "Publish release" button in your Release Draft. Now you can check the pre-release artifacts (e.g. release on [PyPI](https://pypi.org/project/pyprecice/#history)) of the release. No further action is required for a pre-release.
```

Is this actually true? Aren't we just testing the release workflow with a pre-release? We can actually create a pre-release from the release branch that will later be merged into master. This makes perfectly sense.

Reply:
Creating a pre-release from the release branch itself makes sense, we should do this for the future. But until now we never did this.